.github/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# Release 0.20.1-dev
+
+### New features
+
+### Breaking changes
+
+### Improvements
+
+### Bug fixes
+
+generate_hafnian_sample should not pass Qmat as a covariance matrix
```

Comment:
You should add this entry to the existing `Release 0.20.0-dev` below. No need to create a new dev-release. It should also be formatted as a bullet point instead with code references written surrounded by backticks (e.g., `generate_hafnian_samples` and `Qmat`), and a link to this PR. See entries below for examples.